docs/tutorials/quorum-dev-quickstart/using-the-quickstart.md

```diff
@@ -414,6 +415,123 @@ All nodes are validating the same blockchain of transactions, with the private t
+of permissions is specific to only a given node and is configured by placing a file with a list of allowed peers (similar to
+`static-nodes.json`), called `permissioned-nodes.json` in the `data` directory of a GoQuorum node
+* [Enhanced Permissions](../../concepts/permissions-overview.md#enhanced-network-permissioning) is a smart-contract-based
+permissioning model designed for enterprise-level needs and is applicable to all peers on the network.
+
+This example deploys permissioning contracts to the Quickstart and then uses API calls to set appropriate permissions.
+
+The first step is to pick a `GuardianAccount` or an admin account, and we will use the account of Validator1 which has an
+address of `0xed9d02e382b34818e88b88a309c7fe71e65f419d` and we will connect to Validator1's RPC endpoint
+`http://127.0.0.1:21001`
+
+The next step is to compile the contracts and then deploy them in a specific order. The contracts folder
+`cd smart_contracts/permissioning/contracts` has two versions of contracts for reference. With `version 1`,
+the permissioning rules are applied only at the time of transaction entry with respect to the permissioning data
+stored in node memory. With `version 2`, the permissioning rules are applied both at the time of transaction entry
+and block minting with respect to the data stored in the permissioning contracts. This tutorial uses `version 2` of
+the contracts because it is more robust and suited for enterprise use.
+
+The folder contains the following contracts:
+
+* PermissionsInterface.sol : Provide external interface, internal proxy to logical contract
+* PermissionsImplementation.sol : Logic contract, the actual logic of the contract is in this contract
+* OrgManager.sol : Data contract, storage organization related data
+* AccountManager.sol : Data contract, storage account related data
+* NodeManager.sol : Data contract, storage node first shuts down data
+* RoleManager.sol ：Data contract, store data first
+* VoterManager.sol : Data contract, which stores voter data
+
+Navigate to the `smart_contracts/permissioning` directory and then compile the contracts:
+
+```bash
+cd smart_contracts/permissioning
+npm install
+./scripts/compile.sh
+```
+
+Next, deploy the contracts and once complete, execute the `init` function of the `PermissionsUpgradable.sol`
+with the `Guardian account` address that was specified earlier. This has been wrapped up into a single script
+that you can run
+
+```bash
+node ./scripts/deploy.js
+```
+
+When this is completed, an output `permission-config.json` is created which contains the addresses of the
+contract and any account addresses that will serve as admins. In this example we've used all accounts, if you
+deploy these contracts in a production network, please select only those which need to be admins.
+
+The file looks like this (but will have different addresses)
```

Comment:
If you've run the quick start from scratch the address's will be the same!

Reply:
:+1: Fixed that up